source/common/tls/context_impl.cc

```diff
@@ -782,5 +794,15 @@ absl::Status TlsContext::checkPrivateKey(const bssl::UniquePtr<EVP_PKEY>& pkey,
   return absl::OkStatus();
 }
 
+void TlsContextStatsHelper::updateStats() {
+  if (tls_context_.get().cert_chain_ == nullptr) {
+    return;
+  }
+  const absl::optional<uint32_t> days =
```

Comment:
Is this a valid use case when cert chain does not contain "valid until" data? Otherwise do we need to log any warning/error?